src/components/a-th/index.scss

```diff
@@ -0,0 +1,40 @@
+@import "../o-table/mixins/table";
+@import "./mixins/th";
+
+@import "../../styles/trumps/align";
+
+/// Placeholders
+
+%a-th {
+  @include th();
+}
+
+/// Class styles
+
+.a-th {
+  @extend %a-th;
+}
+
+.a-th--dense {
+  @include table-cell--dense();
+}
+
+.a-th--sort {
+  @include th--sort();
+}
+
+.a-th--sort-up {
+  @include th--sort-up();
+}
+
+.a-th--sort-down {
+  @include th--sort-down();
+}
+
+/// Tag Styles
+
+.o-table--all {
```

Comment:
why organism style in an atom? move it from here to the organism

Reply:
I asked myself too where to place it at best.

The case for all table molecules and atoms is the following:
- each components styles itself through classes (recommended)
- it's possible to have styles applied by higher specificity CSS targeting the tagname too, like:
   - `.o-table--all > caption { ... }`
   - `.o-table--all > thead { ... }`
      `.o-table--all > thead > tr { ... }`
      `.o-table--all > thead th { ... }`
      `.o-table--all > thead td { ... }`
   - `.o-table--all > tfoot { ... }`
      `.o-table--all > tfoot > tr { ... }`
      `.o-table--all > tfoot th { ... }`
      `.o-table--all > tfoot td { ... }`
   - `.o-table--all > tbody { ... }`
       `.o-table--all > tbody > tr { ... }`
      `.o-table--all > tbody th { ... }`
      `.o-table--all > tbody td { ... }`
   - `.o-table--all tr { ... }`
      `.o-table--all tr td { ... }`
      `.o-table--all tr th { ... }`
   - `.o-table--all td { ... }`
   - `.o-table--all th { ... }`

Now the question is where to put those styles? 💣 
Either we have all of them duplicated once in the `o-table` organism and once in each molecule/atom.
Or which I'm sure is better for maintainability have them at one place in the regarding molecule or atom.

Reply:
Maintainable is when u are looking for o-table styles to be in o-table. IMHO. Whts the issue bye copy pasting this code to the upper one? just move the code. U dont have to increase specificty or am io wrong?

Reply:
I would love that styles are there were they change the Styling and only have class specicifity. No children selectors even if we have to set a class on the html element. Otherwise we only have problems in future. If u disagree, we can have a talk on friday

Reply:
I'm with you in general, I won't recommend it myself, if I build a single app.

But let's consider that we want to build a holistic product here, in it's pure sense [holistic means](https://www.merriam-webster.com/dictionary/holistic):
> relating to or concerned with wholes or with complete systems rather than with the analysis of, treatment of, or dissection into parts
[from Greek hólos "whole, complete"]

So our data tables shall work with:
- code generated by WYSIWYG editors, like the one from AEM
  - this also includes with or without custom elements, i.e. styles/css only
- any teams using bootstrap and are being familiar with [bootstrap's approach regarding table styling](https://getbootstrap.com/docs/4.0/content/tables/)
- any framework, react, angular, you name it
- of course as custom elements and single CSS class specificity (**recommended**)
- and anything else you can possibly imagine

This holistic approach is the reason I decided to [support Bootstrap's valid approach of table styling](https://getbootstrap.com/docs/4.0/content/tables/), not because I see it the best fit for Apps.

Reply:
Ok lets discuss it on friday evening. I will make a meeting for 17:00. @jackblackCH are u in as well?

Reply:
@LucaMele 
And to make it clear, those styles, do not touch the organism at all.
It's just another why to style molecules and atoms of a table.

Reply:
@LucaMele 
I did a major refactoring, please review again :)